inclusive-design/wecount.inclusivedesign.ca
# fix: fix html validation errors and accessibility alerts in preparation for 2.5.0 release

## Description

Fix html validation errors reported by [the W3C validator](https://validator.w3.org/) and accessibility alerts reported by WAVE.

## Steps to test

1. Run "Initiatives" and "Workshop" pages through [the W3C validator](https://validator.w3.org/);
2. The validator should not report any error;
3. Run every page through WAVE accessibility tool;
4. Errors and contrast errors should be 0. All alerts should be reasonable.

**Expected behavior:** 
See above.
